src/elements/play-pen/play-pen.ts

```diff
@@ -338,6 +372,9 @@ export class PlayPen extends LitElement {
 
     if (props.has('_useRemoteRuntime') || props.has('_remoteRuntimeOrigin'))
       this.#upload()
+
+    if (props.has('_assetFilesystem') && this._assets)
```

Comment:
do we need to check _enableLocalAssets here? I am a little unclear on the distinct from _assetFilesystem.

Reply:
`_enableLocalAssets` allows the user to mount a local directory or a ZIP file as the source for assets.  `_assetFilesystem` is the toggle between the in-browser 'virtual' assets and the 'local' assets.

Reply:
Removed in latest revision